frontend/src/components/project/ProjectContent.js

```diff
@@ -318,38 +327,40 @@ export default function ProjectContent({
             {texts.created}: <DateDisplay date={new Date(project.creation_date)} />
           </Typography>
           <div>
-            <Typography component="span">
-              {texts.started + " "}
-              <TimeAgo
-                date={new Date(project.start_date)}
-                formatter={locale === "de" ? germanYearAndDayFormatter : yearAndDayFormatter}
-              />{" "}
-              {texts.by}
-            </Typography>
-            {project.isPersonalProject ? (
-              <MiniProfilePreview
-                className={classes.creator}
-                profile={project.creator}
-                size="small"
-              />
-            ) : (
-              <MiniOrganizationPreview
-                size="small"
-                className={classes.creator}
-                organization={project.creator}
-              />
-            )}
-            {project.end_date && (
-              <Typography className={classes.finishedDate}>
-                {texts.finished + " "}
+            <div className={classes.projectParentContainer}>
+              <Typography component="span">
+                {texts.started + " "}
                 <TimeAgo
-                  date={new Date(project.end_date)}
+                  date={new Date(project.start_date)}
                   formatter={locale === "de" ? germanYearAndDayFormatter : yearAndDayFormatter}
                 />{" "}
+                {texts.by}
               </Typography>
-            )}
+              {project.isPersonalProject ? (
+                <MiniProfilePreview
+                  className={classes.creator}
+                  profile={project.creator}
+                  size="small"
+                />
+              ) : (
+                <MiniOrganizationPreview
+                  size="small"
+                  className={classes.creator}
+                  organization={project.creator}
```

Comment:
Nit: since these are net new, it'd be great if we could start with them alphabetized for faster readability, and thus maintainability:
```suggestion
                  className={classes.creator}
                  organization={project.creator}
                  size="small"
```

Reply:
I like the suggestion! :)